misc/python/materialize/cli/gen-lints.py

```diff
@@ -176,6 +176,9 @@
     # Implementing `From` gives you `Into` for free, but the reverse is not
     # true.
     "clippy::from_over_into",
+    # Futures get compiled into state machines, which can end up being very large (10's of KB).
+    # It's inefficient to moves these around on the stack, so require that they get boxed.
```

Comment:
It might be useful to include a link to https://github.com/rust-lang/rust/issues/99504 here.